#!/usr/bin/env python
PACKAGE = "omniradar"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("n_sweeps",     int_t,    0, "Number of consectutive sweeps", 1, 1, 50)
gen.add("t_sweep",      double_t, 0, "FMCW signal duration / acquistion time [ms]", 5.0, 0.5, 50.0)
gen.add("config_str",   str_t,    0, "RIC config string",  "06-FE-00-69-60-7A-C0-00-00-00-00-2C-00-00-00-00-01-DC-64")

exit(gen.generate(PACKAGE, "omniradar_node", "omniradar"))
